references/cli/ls_test.go

```diff
@@ -83,6 +84,22 @@ func TestBuildApplicationListTable(t *testing.T) {
 			},
 			expectedErr: nil,
 		},
+		"specified label selector": {
+			app: &v1beta1.Application{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "app1",
+					Namespace: "test",
+					Labels: map[string]string{
+						"app.kubernetes.io/name":    "nginx",
+						"app.kubernetes.io/version": "v1",
+					},
+				},
+				Spec:   componentOrderSpec,
+				Status: componentOrderStatus,
+			},
+			expectedErr:   nil,
+			labelSelector: "app.kubernetes.io/name=nginx,app.kubernetes.io/version=v1",
```

Comment:
I remove this line and find the test passes successfully.

Reply:
> I remove this line and find the test passes successfully.

Because the labelSelector is optional, not mandatory